pkg/apis/pxc/v1/pxc_types.go

```diff
@@ -61,6 +62,15 @@ type PXCScheduledBackup struct {
 	Storages           map[string]*BackupStorageSpec `json:"storages,omitempty"`
 	ServiceAccountName string                        `json:"serviceAccountName,omitempty"`
 	Annotations        map[string]string             `json:"annotations,omitempty"`
+	PITR               PITRSpec                      `json:"pitr,omitempty"`
+}
+
+type PITRSpec struct {
+	Enabled            bool              `json:"enabled"`
+	StorageName        string            `json:"storageName"`
+	Resources          *PodResources     `json:"resources,omitempty"`
+	TimeBetweenUploads int64             `json:"timeBetweenUploads,omitempty"`
+	ImagePullPolicy    corev1.PullPolicy `json:"imagePullPolicy,omitempty"`
```

Comment:
why `ImagePullPolicy` specified on PITR level, but image itself and image pull secrets is part of Backup section?

Reply:
https://github.com/percona/percona-xtradb-cluster-operator/pull/622/commits/aabec733bdf055557998a691d5e502f99c0e9b1f